test/tools/OpenCover/OpenCover.psm1

```diff
@@ -363,7 +363,7 @@ function Install-OpenCover
 {
     param (
         [parameter()][string]$Version = "4.6.519",
-        [parameter()][string]$TargetDirectory = "~/",
+        [parameter()][string]$TargetDirectory = "$home",
```

Comment:
In this case, the quotes don't matter, but it's more consistent with the other parameters.